kategory-docs/docs/docs/datatypes/sequencekw/README.md

```diff
@@ -6,4 +6,61 @@ permalink: /docs/datatypes/sequencekw/
 
 ## SequenceKW
 
-TODO
\ No newline at end of file
+SequenceKW implements lazy lists representing lazily-evaluated ordered sequence of homogenous values.
+
+It can be created from Kotlin Sequence type with a convenient `k()` function.
+
+```kotlin:ank
+import kategory.*
+
+sequenceOf(1, 2, 3).k()
+```
+
+SequenceKW derives many useful typeclasses. For instance, it has a [`Semigroup`](/docs/typeclasses/semigroup/) instance.
```

Comment:
Here we are mixing `Semigroup` with `combineK` which is declared in `SemigroupK`